src/test/java/HomePageTest.java

```diff
@@ -1,4 +1,64 @@
+import org.junit.Test;
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.chrome.ChromeDriver;
+import org.testng.Assert;
+
 public class HomePageTest {
 
 
+    @Test
+    public void LoginSuccess () throws InterruptedException {
+
+        WebDriver driver = new ChromeDriver();
+        driver.get("https://testkwidos.tk/auth/login");
+        driver.manage().window().maximize();
```

Comment:
Please move driver configuration to another method, because you use this configuration in several test methods and also test should follow this AAA pattern: https://cypriops.com/blog/test-food-chain/

Reply:
Ok, thanks